src/Api/PubnubApi/Model/Server/SubscribeMessage.cs

```diff
@@ -28,6 +28,11 @@ internal class SubscribeMessage
         /// </summary>
         private object d { get; set; } = "";
 
+        /// <summary>
+        /// signal indicator
```

Comment:
This is actually a message type indicator

Reply:
corrected "signal" to "message type"